### ISSUE TITLE
Cannot navigate settings by categories

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#92931]
I am using the latest 1.44 insiders on linux. It seems that now I cannot focus the settings tree, the one that allowed me to switch between settings categories. However I do see it in browse mode with orca screenreader, so not sure what has happened.
This didn't happen in previous insiders/in 1.42 definitely, not sure about 1.43 as I am using insiders for some time.

**Comment by roblourens** [comment]
I think this is covered by the investigation in https://github.com/microsoft/vscode/issues/92488

**Comment by webczat** [comment]
not sure. For me I don't see the tree *at all*.

**Comment by webczat** [comment]
Could verify latest insiders

**Comment by roblourens** [comment]
It sounds like an issue in Orca was fixed recently too relating to this, although I can't say whether it's what you're seeing or not

**Comment by webczat** [comment]
confirmed, now I see the tree. This can remain closed.
